eps/README.md

```diff
@@ -0,0 +1,106 @@
+# Enhancement Proposal (EP)
```

Comment:
Feels like this info could live under the `design` directory maybe? Or maybe under `docs/proposals`? Either way, I don't feel like it needs to be its own top level directory.

Reply:
Also, maybe I missed it, but I'm guessing the intention is for someone to create the proposal doc and include it in this directory? Unless it exists already in the doc, be sure to mention that.

Reply:
The gateway-api has a gep folder at the top level which is why I chose this, but I like `docs/proposals`